library/src/main/java/com/schibsted/spain/barista/internal/matcher/DrawableMatcher.kt

```diff
@@ -51,21 +55,10 @@ class DrawableMatcher private constructor(@DrawableRes private val expectedDrawa
       return false
     }
 
-    val bitmap = getBitmap(imageView.drawable)
-    val otherBitmap = getBitmap(expectedDrawable)
-    return if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.HONEYCOMB_MR1) {
-      bitmap.sameAs(otherBitmap)
-    } else {
-      BitmapComparator.compare(bitmap, otherBitmap)
-    }
-  }
+    val bitmap = DrawableToBitmapConverter.getBitmap(imageView.drawable)
+    val otherBitmap = DrawableToBitmapConverter.getBitmap(expectedDrawable)
```

Comment:
"bitmap" and "otherBitmap" are not very descriptive. We should probably give better names explaining what does each bitmap represent.
Right now I can't tell what they are.

Reply:
You approved it previously... back to school: "el profe me tiene manía" 😂

Reply:
> You approved it previously

That sounds like a very _Jose_ excuse!

Reply:
Fix'd!

Reply:
Yes, but it was part of a joke. Stupid bots, you don't have sense of humor! 🤖 

And here's a flag of Romania 🇷🇴, because "el RObot me tiene MANÍA".

... sorry